index.html

```diff
@@ -7,7 +7,122 @@
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
 </head>
+
 <body>
-    <h1>comproDLS</h1>
+    <div class="main" style="width: 100%;">
```

Comment:
Remove inline css. Class name can be snake cased.
Avoid duplicate html for multiple screens. Try to achieve via css / media queries.